src/App.vue

```diff
@@ -6,10 +6,13 @@
       <div id="nav">
         <!-- Build menu based on configured app routes. -->
         <!-- Keep router-link on one line for underlining -->
-        <span v-for="(item, index) in routes" :key="item.path">
+        <span class="nav-item" v-for="(item, index) in routes" :key="item.path">
```

Comment:
This doesn't add any difference I can see so leave it out.
```suggestion
        <span v-for="(item, index) in routes" :key="item.path">
```